### PR TITLE
Update issue template

### PR DESCRIPTION
**Pull request by sosukesuzuki**
## Description

I'm not good at English, so it's a little difficult to read the expected behavior written in text only.

## Checklist

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [ ] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/formatting.md [hunk 1]
```diff
@@ -47,3 +47,5 @@ Don't fill the form below manually! Let a program create a report for you:
 ```
 
 **Expected behavior:**
+
+<!-- Please write both text and code -->
```
Comment (fisker) [inline .github/ISSUE_TEMPLATE/formatting.md:51]: Don't know, but feel hard to understand "text" and "code". Maybe @kachkaev or someone else have a better idea?

Reminder: Please sync this https://github.com/prettier/prettier/blob/e9b8764fc4c770aaa8e5d9b982bda40a83a4cf2a/website/playground/markdown.js#L30 before merge.

Comment (sosukesuzuki) [reply .github/ISSUE_TEMPLATE/formatting.md:51]: Thank you for pointing it out. I would like to know if there is a better wording.

Comment (kachkaev) [reply .github/ISSUE_TEMPLATE/formatting.md:51]: How about something like this?

```suggestion
<!-- Please insert a code snippet and its description -->
```

```suggestion
<!-- Please insert a code snippet and supplement it with a short description -->
```

Comment (fisker) [reply .github/ISSUE_TEMPLATE/formatting.md:51]: Maybe we should add an "Expected output" section?

Comment (kachkaev) [reply .github/ISSUE_TEMPLATE/formatting.md:51]:
````markdown
**Output:**

```jsx
// code snippet
```

**Expected output:**

```jsx
// code snippet
```

**Why?**

<!-- Short explanation -->
````

Comment (sosukesuzuki) [reply .github/ISSUE_TEMPLATE/formatting.md:51]: looks good to me

**Comment by fisker** [comment]
I just realized that we are not using “issue forms” https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

File: website/playground/markdown.js [hunk 2]
```diff
@@ -27,7 +27,10 @@ function formatMarkdown({
     ...(isIdempotent
       ? []
       : ["", "**Second Output:**", codeBlock(output2, syntax)]),
-    ...(full ? ["", "**Expected behavior:**", ""] : []),
+    ...(full ? ["", "**Expected behavior:**", codeBlock(output2, syntax)] : []),
```
Comment (kachkaev) [inline website/playground/markdown.js:30]:
```suggestion
    ...(full ? ["", "**Expected output:**", codeBlock(output2, syntax)] : []),
```

**Comment by sosukesuzuki** [comment]
Playground is broken, I'll investigate it later

**Pull request merged**